### DISCUSSION TITLE
No permission

### DISCUSSION THREAD

**Discussion by qeimada** [fattureincloud/api#132] [Q&A] [answered]
Buongiorno sto migrando da API1 a API2. Ho creato un semplice codice per inviare una nuova fattura. Ottengo questo errore: Exception when calling IssuedDocumentsApi->createIssuedDocument: [403] Client error: `POST https://api-v2.fattureincloud.it/c/*******/issued_documents` resulted in a `403 Forbidden` response: {"error":{"message":"No permission.","code":"NO_PERMISSION"}}

Bisogna dare qualche permesso particolare? Dove sbaglio?

**Comment by emajo** [comment, marked as answer]
Ciao @qeimada, ti consiglio di leggere la guida sugli [scopes](https://developers.fattureincloud.it/docs/basics/scopes), in caso tu stia utilizando il token manuale [qui](https://developers.fattureincloud.it/docs/authentication/manual-authentication#-token-generation) trovi i dettagli a riguardo.

Grazie,
Emanuele.

**Reply by qeimada** [reply]
mi pare di aver creato correttamente il token manuale. E a me serve solo come scope la creazione di una fattura. Ho selezionato il modulo "fatture" dall'elenco. Non basta?

**Reply by valmoz** [reply]
Buongiorno @qeimada 
il messaggio di errore è chiaro. Le consiglio di effettuare le seguenti operazioni:

- Controllare di aver selezionato correttamente il modulo "Fatture", selezionando "Accesso Completo" dalla pagina dell'applicazione FIC
- Controllare che l'access token sia quello legato al collegamento corretto sulla pagina "Applicazioni collegate", e non ad un'altra applicazione a cui sono stati assegnati permessi diversi da quelli attesi
- Se i primi due punti hanno dato esito positivo, provare a scollegare e ricollegare l'applicazione, riassegnando i permessi corretti
- Provare ad utilizzare l'[API Reference](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents) per verificare che il problema non sia legato al suo codice ma ai parametri inseriti
- Leggere la guida di [Invoice Creation](https://developers.fattureincloud.it/docs/guides/invoice-creation) per costruire un body valido

[Qui](https://developers.fattureincloud.it/docs/api-reference) spèieghiamo come usare l'API reference

Ci faccia sapere se in questo modo sarà in grado di risolvere.

Grazie
Mauro

**Reply by qeimada** [reply]
Grazie risolto. Sì era un problema di applicazioni collegate.
